docs/libcurl/symbols-in-versions

```diff
@@ -30,6 +30,7 @@ CURLAUTH_NONE                   7.10.6
 CURLAUTH_NTLM                   7.10.6
 CURLAUTH_NTLM_WB                7.22.0
 CURLAUTH_ONLY                   7.21.3
+CURLAUTH_SIGNATURE_V4           7.72.0
```

Comment:
Why is `V4` and `SIGNATURE` suddenly in the other order in this name? And 7.73.0.